mantidproject/mantid
# Remove Qt3Support Warnings in all other files Group No. 4

This is group 4 of #16129

This fixes the Qt3Support Warnings in all other files Group No. 4

Graph.h     1ebf9e3
Graph3D.cpp     1ebf9e3, 3d9a060
GridDetails.cpp 16583a5
ImageDialog.cpp 16583a5
ImageMarker.cpp 16583a5
ImportASCIIDialog.cpp 41fd77b
ImportWorkspaceDlg.cpp 41fd77b
IntDialog.cpp 41fd77b

_(Changes made which could be looked at carefully can be found next to a comment. The added comments are later removed within these commits - 2c2f987, 51506cf)_

**To Test:**
Code Review
Check MantidPlot builds successfully
Check all builds pass successfully

**Check:**
- Load a workspace and Plot
- Graphs/3D Graphs plotting and working as they should
  - Go to: `Format/Grid` 
  - Play around with the settings and see if it still functions as it should
- Go To `Files / Import ASCII` 
  -  Try importing file and test its functionality





---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
